### PR TITLE
Add to docs

### PR DESCRIPTION
**Pull request by privatezero**
Still in iterative form, but doing some re-arranging and adding of pages/info.

Should (roughly) resolve https://github.com/mipops/dvrescue/issues/28

### CODE REVIEW

**Review by libbyhopfauf — APPROVED** [review]
Looks great to me!

File: docs/_sections/dv_transfer_station.md [hunk 1]
```diff
@@ -1,45 +1,9 @@
 ---
 layout: post
-title: DV Transfer Station
+title: Hardware Setup
```
Comment (ablwr) [inline docs/_sections/dv_transfer_station.md:3]: Do you want to rename the file too?

File: docs/_sections/artifacts.md [hunk 2]
```diff
@@ -0,0 +1,75 @@
+---
+layout: post
+title: Artifacts
```
Comment (ablwr) [inline docs/_sections/artifacts.md:3]: Should this also be "Common DV Artifacts"?

File: docs/_sections/installation.md [hunk 3]
```diff
@@ -0,0 +1,40 @@
+---
+layout: post
+title: Installation
+---
+
+## Installation Guide
+Releases of DVRescue can be downloaded for all supported systems from its page on the [MediaArea website](https://mediaarea.net/DVRescue).
+
+It also is possible to download specific snapshots and daily builds from MediaArea via their download page [here](https://old.mediaarea.net/download/snapshots/binary/dvrescue/).
```
Comment (ablwr) [inline docs/_sections/installation.md:9]: Recommended:

It also is possible to download specific snapshots and daily builds from MediaArea via their [download page](https://old.mediaarea.net/download/snapshots/binary/dvrescue/).

File: docs/_sections/schema_docs.md [hunk 4]
```diff
@@ -0,0 +1,125 @@
+---
+layout: post
+title: DVRescue Schema Documentation
+---
+
+# DVRescue XML Schema Documentation
```
Comment (ablwr) [inline docs/_sections/schema_docs.md:6]: Noiccee

**Review by ablwr — APPROVED** [review]
Great work!! Made a few non-blocking suggestions.

File: docs/_sections/dv_transfer_station.md (hunk 1 above)
Comment (privatezero) [reply docs/_sections/dv_transfer_station.md:3]: not for the time being at least! makes it easier to tie into the non-online docs

File: docs/_sections/installation.md (hunk 3 above)
Comment (privatezero) [reply docs/_sections/installation.md:9]: Don't want to point basic users there for the time being - there are links to the dailies deeper in the docs for people getting more down in the weeds. Can get to snapshots from within MediaArea as well

File: docs/_sections/dv_transfer_station.md (hunk 1 above)
Comment (ablwr) [reply docs/_sections/dv_transfer_station.md:3]: :+1:

File: docs/_sections/artifacts.md (hunk 2 above)
Comment (privatezero) [reply docs/_sections/artifacts.md:3]: wanted to keep less wordy for now - particularly for purpose of sidebar

**Pull request merged**